### PR TITLE
Update websockets client

### PR DESCRIPTION
**Pull request by zSeriesGuy**
Looks like websockets-client returns 2 args in 0.48.0 while in the latest release, it only returns 1 arg.
So I used *args to handle the difference in releases and assume that the last arg is the message.

This should resolve issue #401

### CODE REVIEW

File: plexapi/alert.py [hunk 1]
```diff
@@ -55,7 +55,8 @@ def stop(self):
         log.info('Stopping AlertListener.')
         self._ws.close()
 
-    def _onMessage(self, ws, message):
+    def _onMessage(self, *args):
+        message = args[-1]
```
Comment (pkkid) [inline plexapi/alert.py]: Can you please add a comment here explaining the wonkyness between versions and why we're grabbing the last item? We're all going to forget in about 3 weeks time. :D

File: plexapi/alert.py [hunk 2]
```diff
@@ -65,6 +66,7 @@ def _onMessage(self, ws, message):
         except Exception as err:  # pragma: no cover
             log.error('AlertListener Msg Error: %s', err)
 
-    def _onError(self, ws, err):  # pragma: no cover
+    def _onError(self, *args):  # pragma: no cover
+        err = args[-1]
```
Comment (pkkid) [inline plexapi/alert.py]: Same, lets comment this.

Comment (zSeriesGuy) [reply plexapi/alert.py]: Good point. Done.

**Comment by pkkid** [comment]
Thanks, looks good to me. Going to let others review before merging.

**Comment by Hellowlol** [comment]
I agree :)

**Comment by pkkid** [comment]
Thanks for the fix! :D

**Pull request merged**